src/utils/url.ts

```diff
@@ -82,3 +82,7 @@ export const externalUrlAdditionalProps = {
 
 export const getExtraLinkProps = (isExternal: boolean) =>
   isExternal ? externalUrlAdditionalProps : {};
+
+export const getVenueFullLocation = (venueId: string) => {
+  return new URL(venueInsideUrl(venueId), window.location.origin).href;
+};
```

Comment:
[NB]

```suggestion
export const getFullVenueInsideUrl = (venueId: string) => {
  return new URL(venueInsideUrl(venueId), window.location.origin).href;
};
```

Reply:
@oleksii-front You marked this resolved with out any comment or commit hash. I know that both @mike-lvov and I have asked you not to do this in the past. Please follow our standards.

Reply:
Done in prev commits